ngrinder-controller/src/main/java/org/ngrinder/starter/NGrinderControllerStarter.java

```diff
@@ -261,7 +260,7 @@ public static void main(String[] args) {
 		}
 		final List<String> unknownOptions = commander.getUnknownOptions();
 		final ClusterMode clusterMode = ClusterMode.valueOf(server.clusterMode);
-		clusterMode.parseArgs(unknownOptions.toArray(new String[unknownOptions.size()]));
+		clusterMode.parseArgs(unknownOptions.toArray(new String[0]));
```

Comment:
check if this dump out the unknown  options.

Reply:
It works fine.

according to intellij inspection info late update of openjdk6 the performance of the empty array version the same and sometimes even better.